src/ApiCheck/Baseline/BaselineVisibility.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace ApiCheck.Baseline
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum BaselineVisibility
+    {
+        Public,
+        Protected,
+        Internal,
```

Comment:
I think we can exclude `private`, and then you can add a test that makes sure that something that's actually private just doesn't show up. I could see this generating a lot of noise in various reports.

But it might actually make sense to keep `internal` because of InternalsVisibleTo and cases where we require 100% binary compatibility...